### PR TITLE
examples: improved aural comprehension

### PR DESCRIPTION
**Pull request by telephon**
A bit of restructuring, renaming variables, and an option to use Harry
Partch’s tone system.

## Purpose and Motivation

General Improvement

## Types of changes

A bit of restructuring, renaming variables, and an option to use Harry
Partch’s tone system.

- New feature

## To-do list

- [x] Code is tested
- [x] This PR is ready for review

### CODE REVIEW

**Comment by brianlheim** [comment]
@telephon can you please rebase this on 3.10?

File: examples/research_and_tools/SeminaireMusical.scd [hunk 1]
```diff
@@ -351,23 +461,23 @@ SynthDef("sm_bell", { arg freq=440, amp=0.5;
...
+			min(20000, freq * Rand(3, 2.3) ! 2)
+		).sum;
+
+		//DetectSilence.ar(u, doneAction:2);
+		amp = amp * EnvGen.kr(Env.linen(0, sustain, 0.1), doneAction:2);
+		Out.ar(0,
+			Pan2.ar(u * amp, freq.cpsmidi % 12 / 6 - 1)
+		);
+
+	}).add;
```
Comment (patrickdupuis) [inline examples/research_and_tools/SeminaireMusical.scd]: This SynthDef isn't indented like the rest.

**Comment by telephon** [comment]
@brianlheim Tried, but it was a mess. Sorry, I forgot how this works in our workflow.

**Comment by brianlheim** [comment]
https://github.com/supercollider/supercollider/blob/develop/CONTRIBUTING.md#notes-on-rebasing-and-merge-conflicts

**Comment by telephon** [comment]
Thanks, @brianlheim! Here is a minor improvement (seems to me at least): #4297

**Pull request merged**